src/utils/daos_hdlr.c

```diff
@@ -73,6 +73,14 @@ struct dm_args {
 
 };
 
+/* Report an error with a system error number using a standard output format */
+#define DH_PERROR_SYS(AP, RC, STR, ...)					\
+	fprintf((AP)->errstream, STR ": %s (%d)\n", ## __VA_ARGS__, strerror(RC), (RC))
+
+/* Report an error with a daos error number using a standard output format */
+#define DH_PERROR_DER(AP, RC, STR, ...)					\
+	fprintf((AP)->errstream, STR ": %s (%d)\n", ## __VA_ARGS__, d_errdesc(RC), (RC))
+
```

Comment:
Why `PERROR` instead of `ERROR`? Should we be more similar to the convention of `D_ERROR`? (which doesn't have a `P`)

Reply:
It's quite similar to perror().

This is not a **logging** function and does not use the daos logs at all, it's for reporting program status to the user on stderr.